recipes/treemix/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: treemix
+  version: 1.12
+
+build:
+  number: 0
+  skip: True # [osx]
```

Comment:
You need to add a build string, similar to the salmon recipe. Otherwise, conda won't be able to detect that the package needs a rebuild when CONDA_GSL or CONDA_BOOST are updated.